conf.d/node.d/fronius.conf.md

```diff
@@ -1,3 +1,5 @@
+# Fronius Symo Plugin
+
 [Fronius Symo 8.2](https://www.fronius.com/en/photovoltaics/products/all-products/inverters/fronius-symo/fronius-symo-8-2-3-m)
```

Comment:
redundant info.

Reply:
Need the title. Wanna take it out of the following line (i.e. leaving the 8.2 there?)

Reply:
Can't make the title a link.